src/test/java/core/basesyntax/service/impl/ReportGeneratorImplTest.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ReportGeneratorImplTest {
+    private static final ReportGeneratorImpl reportGenerator = new ReportGeneratorImpl();
+    private static List<String> CORRECT_REPORT_WITH_DATA;
+    private static List<String> REPORT_WITHOUT_DATA;
+
+    @BeforeAll
+    static void setUp() {
+        CORRECT_REPORT_WITH_DATA = List.of("fruit,quantity",
+                "apple,0",
+                "banana,10",
+                "orange,20");
+        REPORT_WITHOUT_DATA = List.of("fruit,quantity");
+    }
+
+    @AfterEach
+    void clearStorage() {

```

Comment:
I suggest to move cleanup logic to BeforeEach, to be sure that db is clear and test data in not affected by earlier changes